modules/formValidation.js

```diff
@@ -37,6 +43,10 @@ const onSubmit = (event) => {
 };
 
 function InitFormValidation() {
+  firstname.addEventListener('input', () => Storage.save('firstname', firstname.value));
+  lastname.addEventListener('input', () => Storage.save('lastname', lastname.value));
+  fullname.addEventListener('input', () => Storage.save('fullname', fullname.value));
+  textArea.addEventListener('input', () => Storage.save('message', textArea.value));
   email.addEventListener('input', () => validateEmail());
   form.addEventListener('submit', (event) => onSubmit(event));
 }
```

Comment:
* When the user loads the page, if there is any data in the local storage the input fields are pre-filled with this data. currently this isn't working please fix it as its part of the project requirements

Reply:
the problem was that we change the function name and we forgot the change it there